app/app.py

```diff
@@ -27,17 +28,21 @@ def home():
 
 @app.route('/predict',methods=['POST'])
 def predict():
-
     inputs = [int(x) for x in request.form.values()]
+    year = datetime.datetime.now().year
+    weekday = datetime.datetime(year=year, month=inputs[2], day=inputs[3]).weekday()
     select_station = request.form.get('station')
     select_station = int(select_station)
     select_line_str = request.form.get('lineID')
     select_line = int(select_line_str)
 
     print(inputs)
-    weatherPrediction = weather.give_prediction(inputs[1], inputs[2], inputs[3], inputs[4])
-    print(weatherPrediction)  # rain, celcius, windGustSpeed, windSpeed, station_name, weather_station, time_of_day
-    inputs.extend(weatherPrediction[0:4])
+    weatherPrediction = weather.give_prediction(inputs[1], inputs[2], inputs[3], inputs[4])[0:4]
```

Comment:
At the moment the app crashes:
"ValueError: Number of features of the model must match the input. Model n_features is 10 and input n_features is 11"

The station name, weather station and time of day from the weather forecast are needed in the print outs on rows 52 and 53 so they should not be deleted here